docs/concepts/guide-data-apis.md

```diff
@@ -53,12 +53,27 @@ Here's the full list of operators supported:
 
 | op  |  Meaning  | Supported types | Examples  |
 |-----|-----------|----------|----------------|
-| eq  | Equals     | integer, string | project_id=eq.pr123 |
+| eq  | Equals     | integer, string, timestamp | project_id=eq.pr123 |
+| neq  | Not Equals     | integer, string, timestamp | amount=neq.531 |
 | lt  | Less than  | integer, string, timestamp | amount=lt.100 |
 | lte | Less than or equal to  | integer, string, timestamp | updated_at=lte.2020-06-01T00:00:00.000-08:00 |
 | gt | Greater than  | integer, string, timestamp | amount=gt.100 |
 | gte | Greater than or equal to  | integer, string, timestamp| updated_at=gte.2020-06-01T00:00:00.000-08:00 |
 | in | Is one of  | integer, string | id=in.(id1,id2,id3) |
+| not_in | Is not one of  | integer, string | creator_user_id=not_in.("usyKlnkmSVw9") |
+| is | Is equals  | null, boolean | is_reimbursable=is.true |
+| is_not | Is not equals  | null, boolean | is_reimbursable=is_not.true |
+| like | Pattern matches with case sensitive  | integer, string | cost_center->name=like.%porate% |
```

Comment:
How about  `Matches case-sensitively with pattern` or something better 
because in all other descriptions, it kind of reads like `x -> operator -> y`. This will keep it similar.

Also, add an example without a subfield and when matching whole i.e. without `%`.